apps/system/js/core.js

```diff
@@ -50,6 +54,80 @@
     },
 
     _start: function() {
+      ScreenManager && ScreenManager.turnScreenOn();
+      // We need to be sure to get the focus in order to wake up the screen
+      // if the phone goes to sleep before any user interaction.
+      // Apparently it works because no other window
+      // has the focus at this point.
+      window.focus();
+      // With all important event handlers in place, we can now notify
+      // Gecko that we're ready for certain system services to send us
+      // messages (e.g. the radio).
+      // Note that shell.js starts listen for the mozContentEvent event at
+      // mozbrowserloadstart, which sometimes does not happen till
+      // window.onload.
+      this.publish('mozContentEvent', {
+        type: 'system-message-listener-ready'
+      }, true);
+
+      this.loadWhenIdle([
+        'CameraTrigger',
+        'NotificationScreen',
+        'AirplaneMode',
+        'NotificationsSystemMessage',
+        'Accessibility',
+        'AlarmMonitor',
+        'DebuggingMonitor',
+        'TimeCore',
+        'GeolocationCore',
+        'TetheringMonitor',
+        'UsbCore',
+        'TextSelectionDialog',
+        'ExternalStorageMonitor',
+        'DeviceStorageWatcher',
+        'AppUsageMetrics',
+        'CellBroadcastSystem', // Blocked by integration test.
```

Comment:
We need this here. B2G integration is testing this but in the design desktop without mobileConnection should not have this functionality.